license.md

```diff
@@ -1,23 +1,7 @@
-Copyright (c) 2015, SilverStripe
-All rights reserved.
+Copyright (c) 2015 Christopher Pitt
```

Comment:
Having said all that, I'm happy to hand the license over if enough people dislike this change.